backend_new/src/enums/shared/geocoding-values.ts

```diff
@@ -0,0 +1,5 @@
+export enum GeocodingValues {
```

Comment:
Curious here why we don't use true/false/unknown types?

Reply:
Can you explain how that would look?

Reply:
Potentially that `verifyRadius` would just return `boolean | unknown`

Reply:
Unfortunately you can't return `unknown` from a function. I was hoping that this GeocodingValues could also be read for the CSV function as well. But I'll keep digging into this